bg_simulator.py

```diff
@@ -58,7 +55,7 @@ def assign_carbs(sugar, last_carbs, sugar_in_range):
         carbs = random.triangular(-50, 100, 60)
     return carbs
 
-def stitch_func(num_days=180):
+def simulate(num_days=10):
```

Comment:
The root problem here is a default in two places. That needs to be fixed somehow. Because of the way you're parsing CLI arguments in a `params` object, you should probably just remove the default here and keep it there.